packages/private-keys/src/ED25519Format.ts

```diff
@@ -0,0 +1,38 @@
+import { generateID, IPrivateKeyFormat, IPrivateKeyStore } from "@tkey/common-types";
+import BN from "bn.js";
+import nacl from "tweetnacl";
+
+export class ED25519Format implements IPrivateKeyFormat {
+  privateKey: BN;
+
+  type: string;
+
+  constructor(privateKey: BN) {
+    this.privateKey = privateKey;
+    this.type = "ed25519";
+  }
+
+  validatePrivateKey(privateKey: BN): boolean {
+    try {
+      nacl.box.keyPair.fromSecretKey(privateKey.toBuffer());
```

Comment:
cannot use toBuffer in the browser. 
Please replace it with `Buffer.from(privateKey.toString("hex", 64), "hex")`
We polyfill buffer for the browser